plugins/inputs/apcupsd/README.md

```diff
@@ -24,11 +24,12 @@ apcupsd should be installed and it's daemon should be running.
 - apcupsd
   - tags:
     - serial
-    - status (string representing the set status_flags)
     - ups_name
+    - status (string representing the set status_flags)
     - model
   - fields:
     - status_flags ([status-bits][])
+    - status
```

Comment:
Hmm having `status` as a field and tag is not ideal. What is the difference in data?

Reply:
There is no difference in data. The disadvantage having the status as tag is that it may change leading to different datasets for the same device. I thought about removing it from tags completely but decided to leave it there for backwards compatibility, avoiding breaking someone's dashboards.

Reply:
thanks for the example, let me chat with the team on Monday and get back to you

Reply:
after chatting with the team it seems historically we have not done this. Instead of someone needs a tag as a field or a field as a tag, then the correct thing is to use the [converter](https://github.com/influxdata/telegraf/tree/master/plugins/processors/converter) processor.

Reply:
Thanks @powersj for your response. So do you suggest to remove the `status` in the current implementation of this PR

1. from the tags - having it as field only (where it should reside IMHO), breaking change
or
2. from the fields - keeping it as a tag, being backwards compatible?

Reply:
> keeping it as a tag, being backwards compatible

This is the way

We try to keep everything as backwards compatible as possible.

Reply:
Well, fine for me. Removed status from fields again.